modules/external-load-balancer/main.tf

```diff
@@ -1,54 +1,61 @@
+resource "google_compute_health_check" "application" {
+  name = "${var.prefix}application"
+
+  check_interval_sec = 5
+  description        = "The TFE application health check."
+  https_health_check {
+    port         = var.port_application_tcp
+    request_path = "/_health_check"
+  }
+  timeout_sec = 4
+}
+
 resource "google_compute_backend_service" "application" {
   health_checks = [google_compute_health_check.application.self_link]
   name          = "${var.prefix}application"
 
   backend {
-    group = var.primary_cluster_endpoint_group_self_link
+    group = var.primary_cluster_instance_group_self_link
 
-    balancing_mode = "RATE"
-    description    = "The backend service for the TFE application."
-    max_rate       = 1000
+    balancing_mode        = "RATE"
+    description           = "The TFE primary cluster group."
+    max_rate_per_instance = 333
```

Comment:
This particular change is from a rate of 1000 for the group to a rate of 333 per instance which I think is almost equivalent since the primaries group is hard-coded to 3 instances. It's not necessary but it provides consistency with the new secondaries backend which must be configured in terms of instances because the group will autoscale.